### ISSUE TITLE
Code Table Request - new catalog record attribute: placental scars

### ISSUE THREAD

**Issue by bryansmclean** [ArctosDB/arctos#6588]
## Goal
Accurately record information about traits of cataloged mammals. This request formalizes agreements by Arctos community members participating in RANGES https://github.com/ArctosDB/arctos/issues/5532. If the requested attribute is a mammal reproductive trait, some discussions began prior to RANGES https://github.com/ArctosDB/arctos/issues/5264.

## Proposed new value. 

placental scars

## Proposed Definition: 

~~Having pigmented areas in the uterus at sites of former embryo implantation.~~

Presence of sites of former embryo implantation in uterus, appearing as pigmented areas.

## Attribute data type: 

categorical [yes/no](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctyes_no)

## Context
_Describe why this new value is necessary and existing values are not._

No current attribute for placental scarring exists. The requested examined/detected value fixes this and has been agreed upon by Arctos members participating in RANGES.

## Table
_Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure._

~~https://arctos.database.museum/info/ctDocumentation.cfm?table=ctexamined_detected~~

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type

## Collection type
_Some code tables contain collection-type-specific values. collection_cde may be found from https://arctos.database.museum/home.cfm_

N/A

## Priority

High

## Example Data
Requests with clarifying sample data are generally much easier to understand and prioritize. Please attach or link to any representative data, in any form or format, which might help clarify the request.

N/A

## Available for Public View

YES

_Project:

## Discussion

 - [x] Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary.

https://github.com/orgs/ArctosDB/teams/arctos-code-table-administrators @dbloom

## Approval
All of the following must be checked before this may proceed.

The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). No person should act in multiple roles; the submitter cannot also serve as a Code Table Administrator, for example.

- [x]  Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ]  Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ]  DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ]  DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection
If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc.

Can a suitable solution be found here? If not, proceed to (2)
Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)
Take the discussion to a monthly Arctos Working Group meeting for final resolution.

## Implementation
 - [ ] Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made.

 - [ ] Review everything one last time. Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel.

Make changes as described above. Ensure the URL of this Issue is included in the definition.

Close this Issue.

DO NOT modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result.

Special Exemptions
In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases.

Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step.
The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology.

**Comment by bryansmclean** [comment]
After [discussions](https://docs.google.com/document/d/1eZ_S8VTp-m5Wrbb6cAK96ODPZpt2-Hvnvt6yfU9qVwg/edit#heading=h.3jdyx487bg21) this attribute should take 3 forms:

repro:placental scar count total
repro:placental scar count left
repro:placental scar count right

**Comment by bryansmclean** [comment]
The group decided to move back to the original request and NOT use ctexamined_detected framework.

**Comment by dustymc** [comment]
OptionA, quantifiable data: https://github.com/ArctosDB/arctos/issues/6588

OptionZ, repro-stuff: https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#reproductive_data

Is this somewhere in the middle option really necessary? There's clearly a cost to having lots of attributes, does the benefit from this justify that cost?

Just for grins, I pulled all 'reproductive data' from Arctos - lots of 'no,' lots of numbers, a few 'unknown,' but I can't find much that looks like "there's scars but I can't/won't count 'em."

[temp_repro.csv.zip](https://github.com/ArctosDB/arctos/files/14105181/temp_repro.csv.zip)

**Comment by Jegelewicz** [comment]
Code Table Committee says this is not needed and is a duplicate of https://github.com/ArctosDB/arctos/issues/6592.
